Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromCondDB.cc

```diff
@@ -159,19 +160,22 @@ GEMEtaPartition* GEMGeometryBuilderFromCondDB::buildEtaPartition(const RecoIdeal
   LogDebug("GEMGeometryBuilderFromCondDB") << "buildEtaPartition " << name << " " << detId;
 
   std::vector<double>::const_iterator shapeStart = rgeo.shapeStart(gid);
-  float be = *(shapeStart + 0) / cm;
-  float te = *(shapeStart + 1) / cm;
-  float ap = *(shapeStart + 2) / cm;
-  float ti = *(shapeStart + 3) / cm;
+  float be = convertMmToCm(*(shapeStart + 0));
+  float te = convertMmToCm(*(shapeStart + 1));
+  float ap = convertMmToCm(*(shapeStart + 2));
+  float ti = convertMmToCm(*(shapeStart + 3));
   float nstrip = *(shapeStart + 4);
   float npad = *(shapeStart + 5);
+  float dphi = *(shapeStart + 6);
```

Comment:
If `dphi` is not in the DB yet, it shouldn't be referenced. Comment out this line until the revised record is uploaded into the DB.

Reply:
Yes, right. I committed it.